src/components/Svg/Svg.tsx

```diff
@@ -0,0 +1,59 @@
+import { PropsWithChildren, SVGProps } from 'react';
+
+import { ClassNameType } from '@/types/util';
+import { tw } from '@/utils/tailwind.util';
+
+type SvgProps = SVGProps<SVGSVGElement> & {
+  /**
+   * width, height와 동시에 적용하는 size입니다. (단위 px)
+   * @default `24`
+   */
+  size?: number;
+
+  /**
+   * size가 존재하더라도 무시하고 적용되는 width입니다. (단위 px)
+   * @default `24`
+   */
+  width?: number;
+
+  /**
+   * height 존재하더라도 무시하고 적용되는 width입니다. (단위 px)
+   * @default `24`
+   */
+  height?: number;
+
+  /**
+   * tailwind으로 svg css 수정할 수 있어요! (크기 조정도 가능합니다)
+   * https://tailwindcss.com/docs/fill
+   * https://tailwindcss.com/docs/stroke
+   * https://tailwindcss.com/docs/stroke-width
+   */
+  className?: ClassNameType;
+};
+
+export const Svg = ({
+  size,
+  width,
+  height,
+  children,
+  className,
+  viewBox,
+  ...rest
+}: PropsWithChildren<SvgProps>) => {
+  const w = width ?? size ?? 24;
+  const h = height ?? size ?? 24;
+  const vb = viewBox ?? `0 0 24 24`;
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width={w}
+      height={h}
+      viewBox={vb}
+      fill="none"
+      className={tw('fill-black', className)}
```

Comment:
아래와 같이 가져가면 좋을 것 같습니다.
이유는 SVG는 모든 아이콘에서 사용되기 때문에 커스텀한 색상이 들어가는 것은 지양되면 좋을 것 같습니다.
색상은 각 아이콘 컴포넌트에서 default로 지정하는 게 추후 많은 아이콘들을 다룰 때 유리할 것 같습니다.
```suggestion
      className={tw(className)}
```

Reply:
기본색깔을 지정한 부분인데, 기본 색깔을 `src/components/Svg`가 아닌 `src/components/Icon/*.tsx`에서 각각 지정하는 의미일까요? 

기본 색상은 흰색 아니면 검은색으로 지정하고 특정 아이콘에서 다른 색깔을 써야한다면, 그 때 tailwind class로 override하려는 의도였습니다!

Reply:
기본 색깔을 'none'으로 가져가는 것으로 생각했어요.
흰색, 검은색, 'none' 셋 중에 검은색으로 기본값 가져가도 괜찮을 것 같습니다.